### DISCUSSION TITLE
Remove hook from magit-status-sections-hook in a repo local .dir-locals.el file

### DISCUSSION THREAD

**Discussion by slowfranklin** [magit/magit#4651] [Support] [answered]
Hi folks!

I have a repository where I have to do large rebases with something like 600 patches in the git rebase todo file. This causes quite some slowdown in magit-status, mainly caused by the magit-insert-rebase-sequence hook.

I've found ways to remove that hook globally in my emacs config, but I'd much prefer to do it via a repo specific .dir-locals.el. Here's what I have which doesn't work:

```
(
 (magit-status-mode
  (remove-hook 'magit-status-sections-hook 'magit-insert-rebase-sequence))
)
```

Anyone with a hint how to get this working? Thanks a lot!

**Comment by tarsius** [comment, marked as answer]
You can use `eval` as documented at https://www.gnu.org/software/emacs/manual/html_node/emacs/Specifying-File-Variables.html.
